containers/base/javascript/browser/css/dashboard.css

```diff
@@ -194,14 +201,23 @@ a{
 }
 
 .dragover{
-    background-color: var(--premium-white-blue);
+    background-color: var(--premium-white-blue) !important;
 }
 
 .sending-interface .cardcontainer{
     display: flex;
     flex-wrap: wrap;
 }
 
+.action{
+    background-color: var(--premium-white) !important;
+    box-shadow: 0 2px 2px 0 rgba(0,0,0,.14), 0 3px 1px -2px rgba(0,0,0,.2), 0 1px 5px 0 rgba(0,0,0,.12);
+}
+
+.action:active{
+    background-color: var(--premium-white-blue) !important;
```

Comment:
`!important` really needed? 🙈

Reply:
Unfortunately needed to overrule material design.